fsspec/implementations/tests/test_http.py

```diff
@@ -80,9 +106,20 @@ def test_list_cache_with_expiry_time_purged(server):
     cached_items = h.dircache.get(server + "/index/")
     assert len(cached_items) == 1
 
+    h.dircache.clear()  # clean up
```

Comment:
Should be fixture, to make sure we clean up even on failure?

Reply:
After the latest update it now uses `tmp_path` which automatically cleans up after runtime.